src/components/Wizard/CreateVmWizard/providers/tests/VCenterVmsWithPrefill.test.js

```diff
@@ -47,25 +62,33 @@ describe('<VCenterVmsWithPrefill />', () => {
     const wrapper = shallow(<VCenterVmsWithPrefill {...props} onChange={onChange} onFormChange={onFormChange} />);
     expect(wrapper).toMatchSnapshot();
   });
-  it('does prefill', () => {
+  it('does prefill', async () => {
     const onChange = jest.fn();
     const onFormChange = jest.fn();
     const wrapper = mount(<VCenterVmsWithPrefill {...props} onChange={onChange} onFormChange={onFormChange} />);
     expect(wrapper).toMatchSnapshot();
     expect(onChange.mock.calls).toHaveLength(0);
     expect(onFormChange.mock.calls).toHaveLength(0);
 
-    wrapper.setProps({ v2vvmware }); // force componentDidUpdate
+    wrapper.setProps({ v2vvmware }); // force componentDidUpdate containing async processing
+    await delay(100);
```

Comment:
Have you tried https://github.com/kubevirt/web-ui-components/blob/master/src/tests/enzyme.js#L3 instead of delay ?

Reply:
done. Thank you, I missed that util function